fixtures/main.py

```diff
@@ -268,11 +282,232 @@ def populate_individuals(db):
+                            "authorizedByIndividualId", ""
+                        ),
+                        "authorizedbyotherid": revision.get("authorizedByOtherId", ""),
+                        "predecessorhash": revision.get("predecessorHash", ""),
+                        "predecessorsignature": revision.get(
+                            "predecessorSignature", ""
+                        ),
+                        "objectdata": revision.get("objectData", ""),
+                        "successorid": revision.get("successorId", ""),
+                        "serializedhash": revision.get("serializedHash", ""),
+                        "serializedsnapshot": revision.get("serializedSnapshot", ""),
+                    }
+                )
+
+                # Update test data
+                data["revisions"][index] = {
+                    "_id": revision.get(
+                        "id", generate_object_id(seed_year + index, 1, 1)
+                    ),
+                    "schemaname": revision.get("schemaname", "dataAgreement"),
+                    "objectid": revision.get("objectid", "1"),
+                    "signedwithoutobjectid": revision.get(
+                        "signedwithoutobjectid", False
+                    ),
+                    "timestamp": revision.get("timestamp", "2024-01-03T09:23:31Z"),
+                    "authorizedbyindividualid": revision.get(
+                        "authorizedbyindividualid", ""
+                    ),
+                    "authorizedbyotherid": revision.get("authorizedbyotherid", ""),
+                    "predecessorhash": revision.get("predecessorhash", ""),
+                    "predecessorsignature": revision.get("predecessorsignature", ""),
+                    "objectdata": revision.get("objectdata", ""),
+                    "successorid": revision.get("successorid", ""),
+                    "serializedhash": revision.get("serializedhash", ""),
+                    "serializedsnapshot": revision.get("serializedsnapshot", ""),
+                }
+            except KeycloakPostError:
+                pass
+
+            index += 1
+    except Exception as e:
+        print(e)
+
+
+def update_organisation_id(db):
+    try:
+        # Get organisation id
+        organisations_collection = db["organizations"]
+        organisation = organisations_collection.find_one()
+        old_organisation_id = organisation["_id"]
+
+        # Delete organisation
+        organisations_collection.delete_one({"_id": organisation["_id"]})
+        # Update organisation id
+        organisation["_id"] = "1"
+        organisations_collection.insert_one(organisation)
+
+        # Update organisation id in policy
+        policies_collection = db["policies"]
+        policy = policies_collection.update_many(
+            {}, {"$set": {"organisationid": organisation["_id"]}}
+        )
+
+        # Update organisation id in users
```

Comment:
Does the individuals collection need to be updated as well ?

Reply:
> Does the individuals collection need to be updated as well ?

Users collection needs to updated because organisation id is present in roles array.

Reply:
> > Does the individuals collection need to be updated as well ?
> 
> Users collection needs to updated because organisation id is present in roles array.

That wasn't my question.

Reply:
> > > Does the individuals collection need to be updated as well ?
> > 
> > 
> > Users collection needs to updated because organisation id is present in roles array.
> 
> That wasn't my question.

Understood, individuals collections are also updated.